.gcb/README.md

```diff
@@ -0,0 +1,14 @@
+# Google Cloud Build support
+
+This directory contains configuration files and scripts to support GCB (Google
+Cloud Build) builds.
+
+We generally prefer GHA (GitHub Actions) for CI (Continuous Integration):
+building the code, running unit test, run any linters or formatters.
+
+We use GCB is better for CD (Continuous Deployment): creating docker images in
```

Comment:
nit: this sentence doesn't make sense; reword

Reply:
Thanks. PTAL.